packages/cache/__tests__/saveCacheV2.test.ts

```diff
@@ -0,0 +1,298 @@
+import * as core from '@actions/core'
+import * as path from 'path'
+import {saveCache} from '../src/cache'
+import * as cacheUtils from '../src/internal/cacheUtils'
+import {CacheFilename, CompressionMethod} from '../src/internal/constants'
+import * as config from '../src/internal/config'
+import * as tar from '../src/internal/tar'
+import {CacheServiceClientJSON} from '../src/generated/results/api/v1/cache.twirp'
+import * as uploadCacheModule from '../src/internal/blob/upload-cache'
+import {BlobUploadCommonResponse} from '@azure/storage-blob'
+
+let logDebugMock: jest.SpyInstance
+
+jest.mock('../src/internal/cacheUtils')
+jest.mock('../src/internal/tar')
+
+beforeAll(() => {
+  process.env['ACTIONS_RUNTIME_TOKEN'] = 'token'
+  jest.spyOn(console, 'log').mockImplementation(() => {})
+  jest.spyOn(core, 'debug').mockImplementation(() => {})
+  jest.spyOn(core, 'info').mockImplementation(() => {})
+  jest.spyOn(core, 'warning').mockImplementation(() => {})
+  jest.spyOn(core, 'error').mockImplementation(() => {})
+  jest.spyOn(cacheUtils, 'getCacheFileName').mockImplementation(cm => {
+    const actualUtils = jest.requireActual('../src/internal/cacheUtils')
+    return actualUtils.getCacheFileName(cm)
+  })
+  jest.spyOn(cacheUtils, 'getCacheVersion').mockImplementation((paths, cm) => {
+    const actualUtils = jest.requireActual('../src/internal/cacheUtils')
+    return actualUtils.getCacheVersion(paths, cm)
+  })
```

Comment:
it seems a bit unnecessary to spy on these methods if we're just returning their actual implementation

can we remove the `jest.mock('../src/internal/cacheUtils')` above and only mock the implementations for what we need?

Reply:
Updated in https://github.com/actions/toolkit/pull/1879/commits/94f18eb26eb16f9d1d79470a9146d4a3cd0cff08